src/appengine/handlers/testcase_detail/show.py

```diff
@@ -368,8 +368,14 @@ def get_testcase_detail(testcase):
   crash_state = testcase.crash_state
   crash_state_lines = crash_state.strip().splitlines()
   crash_type = data_handler.get_crash_type_string(testcase)
-  reproduction_help_url = data_handler.get_reproduction_help_url(
-      testcase, config)
+  formatted_reproduction_help = data_handler.get_formatted_reproduction_help(
```

Comment:
This field needs to be set in configuration.py and also needs to be added in template of configuration page as a multiline text field.

Reply:
Leaving this as-is for now. I think the easy access to help URL is nice since we've decided to keep both. For most use cases this could still work fine.